torch/nn/modules/batchnorm.py

```diff
@@ -54,23 +67,39 @@ class BatchNorm1d(_BatchNorm):
     the mini-batches and gamma and beta are learnable parameter vectors
     of size C (where C is the input size).
 
-    During training, this layer keeps a running estimate of its computed mean
-    and variance. The running sum is kept with a default momentum of 0.1.
+    By default, during training this layer keeps running estimates of its
+    computed mean and variance, which are then used for normalization during
+    evaluation. The running estimates are kept with a default :attr:`momentum`
+    of 0.1.
 
-    During evaluation, this running mean/variance is used for normalization.
+    If :attr:`track_running_stats` is set to ``False``, this layer then does not
+    keep running estimates, and batch statistics are instead used during
+    evaluation time as well.
+
+    .. note::
+        This :attr:`momentum` argument is different from one used in optimizer
+        classes and the conventional notion of momentum. Mathematically, the
+        update rule for running statistics here is
+        :math:`\hat{x}_\text{new} = (1 - \text{momentum}) \times \hat{x}_\text{new} + \text{momemtum} \times x_t`,
+        where :math:`\hat{x}` is the estimated statistic and :math:`x_t` is the
+        new observed value.
 
     Because the BatchNorm is done over the `C` dimension, computing statistics
     on `(N, L)` slices, it's common terminology to call this Temporal BatchNorm
 
     Args:
         num_features: num_features from an expected input of size
-            `batch_size x num_features [x width]`
+            `[batch_size x num_features (x width)]`
```

Comment:
For consistency with convolutions this should be `length` for 1D signals (L for 1D, H x W for 2D, D x H x W for 3D).